graphene/types/mutation.py

```diff
@@ -106,9 +106,7 @@ def __init_subclass_with_meta__(
             else:
                 arguments = {}
         if not resolver:
-            mutate = getattr(cls, "mutate", None)
-            assert mutate, "All mutations must define a mutate method in it"
-            resolver = get_unbound_function(mutate)
+            resolver = get_unbound_function(cls.mutate)
```

Comment:
Not sure that with current implementation it's really needed to call `get_unbound_function()` instead of just referring `cls.mutate`. Should I check it?

Reply:
I think there is no need to use `cls.mutate` as argument to `get_unbound_function` , because it's already store in `mutate` variable. In this case the variable is just being used to validate on assert.